src/main/main-controller.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+    'use strict';
+
+    class MainCtrl {
+      constructor($timeout, currentWindowService) {
+          this.$timeout = $timeout;
+          this.currentWindowService = currentWindowService;
+          currentWindowService.ready(this.onReady.bind(this));
+      }
+      onReady() {
+          this.window = this.currentWindowService.getCurrentWindow();
+      }
+
+      isSummarised() {
```

Comment:
That's fine by me. I just based the names off the assets that Rui made. If the consensus is to change to compact rather than summary, I'm happy to do that.